src/component/Inputcity.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react'
+import Time from './Time'
+import Weatherrr from './Weatherrr'
+const weather = require('openweather-apis');
+
+function Inputcity() {
+    const [cooord, setcooord] = useState({ lati: '51.5085', long: '-0.1257', cityy: 'London' })
+    const [city, setcity] = useState('London')
+    const submit = () => {
+        weather.setLang('en');
+        weather.setCity(`${city}`);
+        weather.setUnits('metric');
+        weather.setAPPID('f7d454f5834522dcb92bcf5d38a1db14');
+        weather.getAllWeather(function (err, info) {
+            if (err) {
+                alert('Input a valid City')
+            }
```

Comment:
the error could be not just an invalid city, so you have to think about them all